packages/jest-expo/src/preset/expoModules.js

```diff
@@ -269,6 +269,7 @@ module.exports = {
           { name: 'clearDiskCache', argumentsCount: 0, key: 'clearDiskCache' },
           { name: 'clearMemoryCache', argumentsCount: 0, key: 'clearMemoryCache' },
           { name: 'prefetch', argumentsCount: 1, key: 'prefetch' },
+          { name: 'getCachePathAsync', argumentsCount: 1, key: 'getCachePathAsync' },
```

Comment:
We autogenerate this JSON during the release, so we don't require contributors to change it 😉 But it's fine, no need to revert it.

Reply:
Oh cool. Wasn't sure how that worked and noticed the bot was complaining about not creating a changelog here despite me seeing the changelog doesn't get updated for every change here. Good to know 🎉